src/Lean/Parser/Term.lean

```diff
@@ -448,6 +448,10 @@ def withAnonymousAntiquot := leading_parser
 @[builtin_term_parser] def «trailing_parser» := leading_parser:leadPrec
   "trailing_parser" >> optExprPrecedence >> optExprPrecedence >> ppSpace >> termParser
 
+/-- Marks a function argument of an `@[extern]`ed function as borrowed.
```

Comment:
```suggestion
/-- Marks a function argument of an function as borrowed. This only has an effect if the function is made available through the FFI using `@[extern]`.
```

I think the use of the attribute is a bit confusing for two reasons: the syntactic noise takes a moment to parse, and "externed" is also not an ordinary word.